### PR TITLE
Adding functionality to Torchvision

### PR DESCRIPTION
**Pull request by NiklasGustafsson**
This PR adds a bunch of new functionality to TV, but still not nms and roi_*.

Problem: I don't know how to **properly** test these, so if anyone has thoughts on that, or would be willing to contribute unit tests, that would be great.

@ChengYen-Tang, @MovGP0, @lostmsu, @kaiidams, @dayo05 -- FYI

### CODE REVIEW

**Comment by NiklasGustafsson** [comment]
As far as NMS and ROI go, I know how to find the binaries and header files, so we can easily include the native code in the build, at least for CPUs. However, it'll take me a bit to figure out how to edit the CMakelist.txt file so that it picks up the right paths and files.

**Comment by ChengYen-Tang** [comment]
It seems that torchvision does not have too many unit tests
https://github.com/pytorch/vision

File: src/TorchVision/Ops/Boxes.cs [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+                }
+
+                if (in_fmt == BoxFormats.xyxy) {
+                    boxes = (out_fmt == BoxFormats.xywh) ? _box_xyxy_to_xywh(boxes) : _box_xyxy_to_cxcywh(boxes);
+                } else if (out_fmt != BoxFormats.xyxy) {
+                    boxes = (in_fmt == BoxFormats.xywh) ? _box_xywh_to_xyxy(boxes) : _box_cxcywh_to_xyxy(boxes);
+                }
+
+                return boxes;
+            }
```
Comment (ChengYen-Tang) [inline src/TorchVision/Ops/Boxes.cs:54]: https://github.com/pytorch/vision/blob/d2f7486ccaef461913cdb51990ff353addf6f064/test/test_ops.py#L1109

File: src/TorchVision/Ops/FrozenBatchNorm2d.cs [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+#nullable enable
+namespace TorchSharp
+{
+    public static partial class torchvision
+    {
+        /// <summary>
+        /// BatchNorm2d where the batch statistics and the affine parameters are fixed
+        /// </summary>
+        public abstract class FrozenBatchNorm2d : nn.Module<Tensor, Tensor>
```
Comment (ChengYen-Tang) [inline src/TorchVision/Ops/FrozenBatchNorm2d.cs:26]: https://github.com/pytorch/vision/blob/d2f7486ccaef461913cdb51990ff353addf6f064/test/test_ops.py#L1044

**Review by ChengYen-Tang — COMMENTED** [review]
https://github.com/pytorch/vision/blob/d2f7486ccaef461913cdb51990ff353addf6f064/test/test_ops.py

File: src/TorchVision/Ops/FrozenBatchNorm2d.cs (hunk 2 above)
Comment (NiklasGustafsson) [reply src/TorchVision/Ops/FrozenBatchNorm2d.cs]: Thanks for the pointers, @ChengYen-Tang. I've incorporate unit tests for the new functionality now.

**Pull request merged**